mautic/mautic
# Fix reply for email with suffix

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If you use email with suffix for example `name+alias@gmail.com` then contact replies does not work properly  https://docs.mautic.org/en/channels/emails/contact-replies



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Setup contact replies monitoring https://docs.mautic.org/en/channels/emails/contact-replies
3. Create campaign with send email and reply decision
4. Send campaign email to contact with email address with suffix like `name+alias@gmail.com`
5. Then reply to the email
6. Then run `php bin/console mautic:email:fetch`
7. Check If reply was triggered properly
